### PR TITLE
USWDS - Color: Includes emergency colors in state tokens

### PR DESCRIPTION
**Pull request by gscottqueen**
## Preview

[Site preview state](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/gscottqueen-gsq-updates-emergency-color-tokens/design-tokens/color/state-tokens/)
[Site preview settings](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/gscottqueen-gsq-updates-emergency-color-tokens/documentation/settings/#color-settings)

## USWDS - Color: Includes emergency colors in state tokens

closes USWDS - Color: Document emergency colors on site. #1139

## Description

- includes an update to the state colors intro descriptions in response to the new value tokens
- included in state tokens and settings color yml

## Additional information

Screen Shot 2021-02-01 at 11 17 54 AM

Screen Shot 2021-02-01 at 12 00 52 PM

### CODE REVIEW

**Comment by gscottqueen** [comment]
@mejiaj tagging you here for a  review -

**Review by mejiaj — APPROVED** [review]
I agree with the change to add them to state tokens makes more sense. Thanks!

File: _data/settings/color.yml [hunk 1]
```diff
@@ -416,6 +416,23 @@ contents:
...
+    description: Colors of emergency actions.
+  - name: emergency
+    subsection: state color token
+    description: Use grade 60v.
+    var: $theme-color-emergency
+    default: '"red-warm-60v"'
+    type: color
+  - name: emergency-dark
+    subsection: state color token
+    description: Use grade 80.
```
Comment (thisisdano) [inline _data/settings/color.yml:430]: Remove description

```suggestion
    description:
```

File: _data/settings/color.yml [hunk 2]
```diff
@@ -416,6 +416,23 @@ contents:
     default: '"blue-cool-60"'
     type: color
 
+  - name: Emergency colors
+    section: true
+    description: Colors of emergency actions.
+  - name: emergency
+    subsection: state color token
+    description: Use grade 60v.
```
Comment (thisisdano) [inline _data/settings/color.yml:424]: Remove description

```suggestion
    description:
```

**Review by thisisdano — CHANGES_REQUESTED** [review]

File: _data/settings/color.yml (hunk 1 above)
Comment (gscottqueen) [reply _data/settings/color.yml]: removed those descriptions

File: _data/settings/color.yml (hunk 2 above)
Comment (gscottqueen) [reply _data/settings/color.yml]: 👍

**Comment by thisisdano** [comment]
Looks good. Ignoring Snyk as irrelevant for this PR.

**Pull request merged**